src/components/Button/Button.scss

```diff
@@ -126,7 +126,86 @@ $default-active-shadow: 0 3px 4px 0 rgba($black, 0.3);
     font-size: 16px;
     padding: 8px 32px;
   }
+
+  &--loading {
+    position: relative;
+    color: transparent;
+    &:hover,
+    &:active,
+    &:focus {
+      color: transparent;
+    }
+
+    .button--indicator {
```

Comment:
can we achieve the same with a pseudo-element? `:before`? To avoid having to add a new element

Reply:
Yeah I had that in there before I pushed but I wasn't sure if any other subclasses of button would have their own rules for ```:before``` or ```:after```. If you think thats safe I can definitely remove the element and change the rule.

Reply:
Yes, it should be. All button styles are on Buttons.scss

Reply:
Sounds good 👍